schema/artifact-manifest-schema.json

```diff
@@ -0,0 +1,29 @@
+{
+  "description": "OpenContainer Artifact Manifest Specification",
+  "$schema": "https://json-schema.org/draft-04/schema#",
```

Comment:
I don't have much experience with json-schema, but had gotten a ping when I previously switched this to https and there's a PR pending to fix that back to http in the other manifests: https://github.com/opencontainers/image-spec/pull/931

Reply:
Thanks for pointing that out. Can keep this comment open and revert it once the other one is reverted.